pkg/generator/ingress_translator_test.go

```diff
@@ -1703,6 +2009,42 @@ var (
 			certificates.CaCertName: cert,
 		},
 	}
+	validCAConfigmap = &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "knative-testing",
+			Name:      "valid-ca",
+			Labels: map[string]string{
+				informerfiltering.KnativeCABundleLabelKey: "true",
+			},
+		},
+		Data: map[string]string{
+			certificates.CaCertName: string(cert),
```

Comment:
Maybe we should have multiple cas to test against?

Reply:
👍  I'll do that.